mobile/src/bundles/talent/store/actions.ts

```diff
@@ -14,4 +17,13 @@ const completeProfileStep = createAsyncThunk<
     return talentApi.completeProfileStep(profileStepPayload);
 });
 
-export { completeProfileStep };
+const completeBadgesStep = createAsyncThunk<
```

Comment:
Let's try to reuse completeProfileStep, I believe we can have only 1 action for each step

Reply:
Tatiana make this changes in https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/341